Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter_test.py

```diff
@@ -219,6 +213,26 @@ def test_get_indicator_context(client, requests_mock, indicator_type, indicator_
         assert result.to_context != []
 
 
+@pytest.mark.parametrize(
+    "indicator_type, indicator_value, json_test_file",
+    [
+        ("ipv4-addr", "1.1.1.1", "test_data/indicator_context_unknown.json"),
+    ],
+)
+def test_get_indicator_context_unknown_indicator(client, requests_mock, indicator_type, indicator_value, json_test_file):
+    mock_response = util_load_json(json_test_file)
+    requests_mock.get(
+        MOCK_URL + f"/v2/inthreat/indicators/context?value={indicator_value}&type={indicator_type}",
+        json=mock_response,
+    )
+
+    args = {"value": indicator_value, "type": indicator_type}
+    command_results = SEKOIAIntelligenceCenter.get_indicator_context_command(client=client, args=args)
+    for result in command_results:
+        assert result.outputs != []
```

Comment:
Please add assertion for the command_results[0] values

Reply:
Done thank you :pray: